src/api/spec/components/watchlist_component_spec.rb

```diff
@@ -80,6 +80,17 @@
     end
   end
 
+  context 'when dealing with new records' do
+    let(:project) { Project.new }
```

Comment:
This could have been `let(:project) { build(:project, name: nil) }`, but we found this not as clear as relying on `Project.new`.